### DISCUSSION TITLE
401 on connecting to firefly

### DISCUSSION THREAD

**Discussion by jschulman** [victorbalssa/abacus#27] [Bug]
Setting up Abacus to connect to my firefly instance.  Here are the server logs when I try to connect:

172.31.0.10 - - [02/Aug/2022:19:38:51 -0500] "POST /login HTTP/1.1" 302 3530 "-" "Mozilla/5.0 (iPhone; CPU iPhone OS 16_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Mobile/15E148 Safari/604.1"
172.31.0.10 - - [02/Aug/2022:19:38:52 -0500] "GET /oauth/authorize?client_id=43&client_secret=xxx&code_challenge=xxx&code_challenge_method=S256&redirect_uri=abacusiosapp%3A%2F%2Fredirect&response_type=code&state=AHlE11up1y HTTP/1.1" 200 3578 "-" "Mozilla/5.0 (iPhone; CPU iPhone OS 16_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Mobile/15E148 Safari/604.1"
172.31.0.10 - - [02/Aug/2022:19:38:58 -0500] "POST /oauth/authorize HTTP/1.1" 302 2598 "-" "Mozilla/5.0 (iPhone; CPU iPhone OS 16_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.0 Mobile/15E148 Safari/604.1"
172.31.0.10 - - [02/Aug/2022:19:38:59 -0500] "POST /oauth/token HTTP/1.1" 200 2733 "-" "Abacus/0.1.0 CFNetwork/1388 Darwin/22.0.0"
172.31.0.10 - - [02/Aug/2022:19:39:02 -0500] "GET /api/v1/about/user HTTP/1.1" 401 956 "-" "Abacus/0.1.0 CFNetwork/1388 Darwin/22.0.0"

Authentication works correctly.  You can see the redirect set right.  Last GET request gets a 401.

Firefly Version [5.7.10]
IOS16

**Comment by victorbalssa** [comment]
- Do you have a `/` at the end of the backend URL? If so, try to remove it,
- After manually requesting an Oauth2 token with postman, can you retry to `GET /api/v1/about/user` to see if it works? https://learning.postman.com/docs/sending-requests/authorization/#requesting-an-oauth-20-token
- try to download the latest version 0.2.1
